mslib/msui/qt5/ui_multiple_flightpath_dockwidget.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Form implementation generated from reading ui file 'mslib/msui/ui/ui_multiple_flightpath_dockwidget.ui'
+# Form implementation generated from reading ui file '/home/zandar/Desktop/MSS/mslib/msui/ui/ui_multiple_flightpath_dockwidget.ui'
```

Comment:
normally there is a relative path only, have you used some options?

Reply:
Same question